cron/monthly/collective-report.js

```diff
@@ -142,7 +143,7 @@ const processCollective = async collective => {
   ];
 
   let emailData = {};
-  const options = { attachments: [] };
+  const options = { attachments: [], roles: [roles.ADMIN, roles.ACCOUNTANT] };
```

Comment:
The property is in the singular because it matches the query interface of the model `role` property.
```suggestion
  const options = { attachments: [], role: [roles.ADMIN, roles.ACCOUNTANT] };
```

Shouldn't TS catch this?

Reply:
Good catch, updating it now. TS wouldn't catch this because we're in a JS file